src/timeseriesflattener/flattened_dataset.py

```diff
@@ -303,6 +304,24 @@ def _drop_records_outside_interval_days(
             axis=1,
         )
 
+    @staticmethod
+    def _embed_text_values(
+        df: pd.DataFrame,
+        embedding_fn: Callable,
```

Comment:
Ideally, we would specify this as an abstract class to make the interface more explicit, making it easier to add new embedding functions.

Reply:
I've specified the format in the docstring now (quite simple: pd.Series with text in, pd.DataFrame out), and added some example embedding functions. I think ABC would be a bit overkill in this case (similar to resolve_multiple_fns)